gradle-plugin/it/src/it/triggersannotation/expected/kubernetes.yml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: v1
+kind: List
+items:
+- apiVersion: apps/v1
+  kind: ReplicaSet
+  metadata:
+    annotations:
+      jkube.io/git-url: "@ignore@"
+      jkube.io/git-commit: "@ignore@"
+      jkube.io/git-branch: "@ignore@"
+      image.openshift.io/triggers: |-
+        [ {
+          "from" : {
+            "kind" : "ImageStreamTag",
+            "name" : "triggersannotation:latest"
+          },
+          "fieldPath" : "spec.template.spec.containers[?(@.name==\"orgeclipsejkubeintegrationtestsgradle-triggersannotation\")].image"
+        } ]
```

Comment:
We need to discuss if this enricher should be moved to the OpenShift specific section and removed from the default Kubernets plugins profile